core/src/test/scala/org/apache/spark/sql/crossdata/catalyst/catalog/persistent/zookeeper/ZookeeperCatalogIT.scala

```diff
@@ -38,8 +46,50 @@ class ZookeeperCatalogIT extends BaseXDTest /* extends SharedXDSession*/ {
     val tables = new TableDAO(catalogConfig)
   }
 
-  "A Zookeeper persistent catalog" should "persist entries" in {
-    ()
+  protected def cleanPersistence(): Unit = Seq(DAO.dbs.dao, DAO.tables.dao) foreach (_.deleteAll)
+
+  override protected def beforeAll(): Unit = cleanPersistence()
```

Comment:
That's intentionally done so. If for some reason the test executed addressing a non-volatile zookeeper server and its process was interrupted before executing `afterAll`, then the next execution would fail or, what is worst, not-file even though it started in an inconsistent state.